gradle/asciidoc.gradle

```diff
@@ -74,5 +80,67 @@ configure(rootProject) {
 		}
 		from(asciidoctor) { into("docs/") }
 	}
+
+	configurations {
+		adoc
+	}
+
+	dependencies {
+		//FIXME add to version catalog
```

Comment:
mmh I should probably address that one right away, and make sure the latest milestone is in use (must be M7 now)